Templates/templates/PHP/Model/ComplexType.php.tt

```diff
@@ -41,9 +41,11 @@ if (complex.IsBaseAbstractAndReferencedAsPropertyType() && !complex.IsAbstract)
 #>    /**
     * Set the @odata.type since this type is immediately descended from an abstract
     * type that is referenced as the type in an entity.
+    * @param array $propDict The property dictionary
     */
-    public function __construct()
+    public function __construct($propDict = array())
```

Comment:
adding parameters with default values to existing methods/ctors in dotnet is compatible from a prototype standpoint but constitutes a binary breaking change. I just want to double check this is not also the case in PhP.
https://docs.microsoft.com/en-us/dotnet/core/compatibility/
https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/breaking-changes#binary-breaking-change

Reply:
I would argue that the code wasn't working in the first place and adding this wouldn't require any code changes on the end of the client to make the new code work.

Reply:
thanks for confirming